applications/asset_tracker_v2/overlay-lwm2m.conf

```diff
@@ -13,6 +13,7 @@ CONFIG_LWM2M_RW_JSON_SUPPORT=n
 CONFIG_LWM2M_SECURITY_INSTANCE_COUNT=3
 CONFIG_COAP_INIT_ACK_TIMEOUT_MS=4000
 CONFIG_BASE64=y
+CONFIG_LWM2M_RD_CLIENT_SUSPEND_SOCKET_AT_IDLE=y
```

Comment:
Move this below `CONFIG_LWM2M_QUEUE_MODE_ENABLED=y` in line 37 since this KConfig is only applicable when Queue mode is enabled.

Reply:
Alright